tools/authz.env

```diff
@@ -0,0 +1,7 @@
+JWT_SECRET=Jukebox
```

Comment:
Starting to have many files like these...
prisma, this one, the one at the root...

Just sayin... :D

Reply:
@julienstroheker What do you suggest ?

Reply:
to merge this and we can figure this out later 🚀